src/lib/registration/url_helpers.rb

```diff
@@ -119,6 +120,13 @@ def self.reg_url_at_installation
       slp_service_url
     end
 
+    # get registration URL from AutoYaST configuration file
+    def self.reg_url_from_autoyast_config
+      server = ::Registration::Storage::Config.instance.reg_server
```

Comment:
Yes, that's right, but while calling
Helpers::write_conifg  the function UrlHelpers.registration_url will be called and returns nil

2015-08-27 15:45:04 <2> 10.20.90.152(2926) [Ruby] registration/url_helpers.rb:73 Unknown mode: autoinstallation, using default URL
2015-08-27 15:45:04 <1> 10.20.90.152(2926) [Ruby] registration/helpers.rb:100 Boot reg_ssl_verify option: nil
2015-08-27 15:45:04 <1> 10.20.90.152(2926) [Ruby] registration/helpers.rb:134 writing registration config: {:url=>nil, :insecure=>false}

The interesting part is, that 
registration/url_helpers.rb:73 Unknown mode: autoinstallation, using default URL
will be logged only while calling the write_config function.